### PR TITLE
Simplify Development Process

### PR DESCRIPTION
**Pull request by nathunsmitty**
There's this cool package called [concurrently](https://github.com/kimmobrunfeldt/concurrently) that lets you run multiple commands in package.json. Since we have to `npm start` and `grunt build`, I figured it'd be easier to just consolidate both commands. Check it out: you only have to run `npm start`!

### CODE REVIEW

**Comment by nathunsmitty** [comment]
I think an issue with putting our new script under start is that we don't want to be running nodemon and concurrently in production, which I think just calls `npm start`. Not sure about this though, and will look into it more.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ This is the complete code for UCLA Radio's website, Node.js server and API. We a
 UCLA Radio is a completely student-run organization offering cultural content created by 100+ DJs. [Let us know](mailto:radio.web@media.ucla.edu) if you have any suggestions! All of the software here is written by students.
 
 How to run locally:
-* Ensure you have Mongo installed and a Mongo db instance is running
+* Ensure you have Mongo and Node installed
   * Mac: run `setup.sh` on your Mac with [homebrew](http://brew.sh/) installed to set up node and MongoDB automatically
+* Run the MongoDB instance in a separate terminal instance with `mongod`
```
Comment (nathunsmitty) [inline README.md]: Should we just add `mongod` as a third running task?

**Comment by binerys** [comment]
Added mongod to the process! I also don't think we can name it dev because I ran into the following issue: 

image

Doesn't look like it recognizes it. 

Also this branch is rebased with master so there shouldn't be any merge conflicts

**Comment by nathunsmitty** [comment]
Oh you have to run `npm run dev`! You can `npm run <script-name>` for any script defined in the `"scripts"` object.

**Comment by binerys** [comment]
Cool! Changed it back to "dev". This is ready to merge in whenever!

**Comment by nathunsmitty** [comment]
Alright, I just updated the readme, add `npm start` (in case we ever want that), and readded the SQL line. (We can handle that in the readme-update branch)

**Pull request merged**